.github/workflows/dev-testing.yml

```
@@ -51,6 +51,8 @@ jobs:
         run: pip install --no-cache-dir git+https://github.com/dandi/dandi-cli@master
       - name: Dev gallery - PyNWB
         run: pip install --no-cache-dir git+https://github.com/NeurodataWithoutBorders/pynwb@dev
+      - name: Dev gallery - ProbeInterface
+        run: pip install --no-cache-dir git+https://github.com/spikeinterface/probeinterface@main
       - name: Dev gallery - SpikeInterface
         run: pip install --no-cache-dir git+https://github.com/spikeinterface/spikeinterface@main
```

```suggestion
        run: pip install --no-cache-dir git+https://github.com/spikeinterface/spikeinterface@main[test_core]
```

Let's also add this so we don't have an implicit requirement that is only reflected on the order of the installation. `test_core` should reflect that is required to run spikeinterface as that is what is the basic package tested with:

https://github.com/SpikeInterface/spikeinterface/blob/74abe6c53b08a46ecb42570c741aa3f773e0988e/pyproject.toml#L116-L125